docs/ref/models/querysets.txt

```diff
@@ -1793,10 +1793,10 @@ them::
     <QuerySet [<Person: ...)>, ...]>
 
 Currently, the ``postgresql``, ``oracle``, and ``mysql`` database
```

Comment:
`Currently, the` → `The`

Reply:
IMO, we would leave `Currently` because `FOR UPDATE` may be implemented in SQLite in the future :thinking:

Reply:
It just seemed redundant as it is stating the backends that support it explicitly. I wouldn't expect that to be anything other than the present tense, i.e. current.

It's only a minor thing, so feel free to skip it.